lib/teracy-dev-essential/config/rsync_recovery.rb

```diff
@@ -47,6 +47,10 @@ def configure_node(settings, config)
           end
         end
 
+        if TeracyDev::Util.require_version_valid?(@vagrant_version, ">= 2.2.4") and TeracyDev::Util.require_version_valid?(@vagrant_version, "< 2.2.7")
```

Comment:
we force to use rsync-auto on vagrant 2.2.5 and 2.2.6, so use `>= 2.2.5` and `< 2.2.7` instead

Reply:
add logger.info
```ruby
@logger.info("gatling-rsync-auto does not work on vagrant 2.2.5 and 2.2.6, so going to use rsync-auto instead")
```